tests/python_on_whales/components/test_compose.py

```diff
@@ -894,6 +894,37 @@ def test_compose_logs_follow():
     docker.compose.down(timeout=1)
 
 
+def test_events():
+    docker = DockerClient(
+        compose_files=[
+            PROJECT_ROOT
+            / "tests/python_on_whales/components/compose-with-healthcheck.yml"
+        ],
+        compose_compatibility=True,
+    )
+
+    docker.compose.up(["unhealthy_service"], detach=True)
+
+    events_iterator = docker.compose.events(["unhealthy_service"])
+
+    events = []
+    try:
+        for event in events_iterator:
+            events.append(event)
+            if len(events) >= 3:  # Collect a few events for testing
+                break
+    except KeyboardInterrupt:
+        pass
+
+    assert len(events) > 0
+    assert all(
+        isinstance(event, ComposeEvent) and event.service == "unhealthy_service"
+        for event in events
+    )
+

```

Comment:
Could you check a bit more info? Like the other attributes for example, and the number of event collected (we make sure the for loop actually ran). Notably, it would make sure the other attributes are not `None`, and that we didn't make any typos in the pydantic model for the events.